### ISSUE TITLE
Will there be future updates?

### ISSUE THREAD

**Issue by IdotMaster1** [qtinsider/meegram2#5]
At least some documentation on how to build for other developers that want to collaborate

**Comment by qtinsider** [comment]
Absolutely, we understand the importance of clear documentation for collaboration. We'll work on providing comprehensive build instructions to make it easier for developers to collaborate effectively. Thank you for raising this! 📚👩‍💻

**Comment by IdotMaster1** [comment]
Thanks!
